### ISSUE TITLE
UI/UX Improvements to Project Overview Page (processed from Tech Support form to oli-help)

### ISSUE THREAD

**Issue by kwahlenmayer** [Simon-Initiative/oli-torus#3635]
UI/UX[Michael Ringenberg]
Created by OLI Help
M
[Michael Ringenberg](https://olihelp.freshdesk.com/a/contacts/32006031141) reported via the portal
4 days ago (Fri, 2 Jun 2023 at 10:13 AM)
 
On 2023-06-02 14:13:11.127353+00:00 UTC UTC, Michael Ringenberg < mringenb@andrew.cmu.edu > wrote:

On the Project Overview page when the browser is in full screen mode (Windows 11, chrome), the first couple of letters of the "Cookie Preferences" link label are obscured by the side bar navigation.

----------------------------------------------
Timestamp: 2023-06-02 14:13:11.127353+00:00 UTC UTC
Ip Address: 0.0.0.0.0.65535.32512.1
Location: https://proton.oli.cmu.edu/authoring/project/ringenberg_demo

**Comment by kwahlenmayer** [comment]
I was able to replicate this by turning on full screen and zooming in several times. Then I turned off full screen and still could not see the beginning of "cookie preferences" on the bottom left of the Project Overview page though eventually I zoomed out enough to see "cookie preferences".
https://github.com/Simon-Initiative/oli-torus/assets/24357828/563d095e-4177-42c3-92e1-00c0a9629e1b

Replicate by zooming in enough on the Project Overview page to not be able to see all of the word Cookie in the bottom left.

I further replicated this by opening another browser (Firefox) and was not able to see Cookie Preferences when at 100% zoom. Zooming to 90% allowed all of Cookie Preferences to be seen.
